cypress/e2e/slices/02-simulator-screenshots.cy.js

```diff
@@ -0,0 +1,83 @@
+import { SlicePage } from "../../pages/slices/slicePage";
+import { SimulatorPage } from "../../pages/simulator/simulatorPage";
+import { ScreenshotModal } from "../../pages/simulator/screenshotModal";
+
+describe("I am an existing SM user and I want to take a screenshot from the slice simulator", () => {
+  const random = Date.now();
+
+  const slice = {
+    id: `test_screenshots${random}`,
+    name: `TestScreenshots${random}`,
+    library: "slices",
+    newVariationName: "foo",
+  };
+
+  const slicePage = new SlicePage();
+  const simulatorPage = new SimulatorPage();
+  const screenshotModal = new ScreenshotModal();
+
+  before("Cleanup local data and create a new slice", () => {
+    cy.clearProject();
+    cy.setSliceMachineUserContext({});
+    cy.createSlice(slice.library, slice.id, slice.name);
+
+    // add a variation
+    cy.get("button").contains("Default").click();
```

Comment:
In case you didn't notice there is a `addVariationToSlice()` helper in https://github.com/prismicio/slice-machine/blob/master/cypress/helpers/slices.js#L120 
But I see that in your version you input a different variation id so it might suit your use case